### PR TITLE
[nrf noup] ED25519 Defaults for nrf54l15dk

### PR DESCRIPTION
**Pull request by de-nordic**
Kconfig defaults required for ED25519 on nrf54l15.

### CODE REVIEW

File: boards/nordic/nrf54l15pdk/Kconfig.defconfig [hunk 1]
```diff
@@ -10,6 +10,23 @@ config ROM_START_OFFSET
 	default 0 if PARTITION_MANAGER_ENABLED
 	default 0x800 if BOOTLOADER_MCUBOOT
 
+# What MCUboot requires is here
+if MCUBOOT
+
+# Within sdk-nrf NRF security is default
+config NRF_SECURITY
+	bool
```
Comment (nordicjm) [inline boards/nordic/nrf54l15pdk/Kconfig.defconfig:18]:
```suggestion
```

File: boards/nordic/nrf54l15pdk/Kconfig.defconfig [hunk 2]
```diff
@@ -10,6 +10,23 @@ config ROM_START_OFFSET
...
+	default y
+	select NRF_OBERON
+
+# NRF_SECURITY enforces PSA crypt
+config MBEDTLS
+	bool
```
Comment (nordicjm) [inline boards/nordic/nrf54l15pdk/Kconfig.defconfig:24]:
```suggestion
```

**Review by nordicjm — CHANGES_REQUESTED** [review]
Do not re-declare already existing symbols

File: boards/nordic/nrf54l15pdk/Kconfig.defconfig (hunk 1 above)
Comment (de-nordic) [reply boards/nordic/nrf54l15pdk/Kconfig.defconfig]: Fixed

File: boards/nordic/nrf54l15pdk/Kconfig.defconfig (hunk 2 above)
Comment (de-nordic) [reply boards/nordic/nrf54l15pdk/Kconfig.defconfig]: Fixed

**Review by nvlsianpu — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]

**Review by nordicjm — CHANGES_REQUESTED** [review]
PDK is no longer supported `NRF_SECURITY` if needed by MCUboot should be selected by Kconfig in MCUboot

File: boards/nordic/nrf54l15dk/Kconfig.defconfig [hunk 3]
```diff
@@ -12,6 +12,20 @@ config ROM_START_OFFSET
 
 endif # BOARD_NRF54L15DK_NRF54L15_CPUAPP
 
+# What MCUboot requires is here
+if MCUBOOT
+
+# Within sdk-nrf NRF security is default
+config NRF_SECURITY
+	default y
+	select NRF_OBERON
```
Comment (ArekBalysNordic) [inline boards/nordic/nrf54l15dk/Kconfig.defconfig:21]: Why do we need to use SW crypto? Why can't we use CRACEN? Does ED25519 require something that CRACEN cannot process?

Comment (de-nordic) [reply boards/nordic/nrf54l15dk/Kconfig.defconfig:21]: No, that is mistake.

**Pull request closed without merging**